front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -0,0 +1,158 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import {
+  electionReportCodes,
+  F3xReportCode,
+  F3xReportCodes,
+  monthlyElectionYearReportCodes,
+  monthlyNonElectionYearReportCodes,
+  quarterlyElectionYearReportCodes,
+  quarterlyNonElectionYearReportCodes,
+  quarterlySpecialReportCodes,
+} from 'app/shared/models/f3x-summary.model';
+import { LabelList, LabelUtils, PrimeOptions, StatesCodeLabels } from 'app/shared/utils/label.utils';
+import { selectCommitteeAccount } from 'app/store/committee-account.selectors';
+import { ValidateService } from 'app/shared/services/validate.service';
+import { schema as f3xSchema } from 'fecfile-validate/fecfile_validate_js/dist/F3X';
+import { Subject, takeUntil } from 'rxjs';
+import { Store } from '@ngrx/store';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { environment } from 'environments/environment';
+
+@Component({
+  selector: 'app-create-f3x-step1',
+  styleUrls: ['./create-f3x-step1.component.scss'],
+  templateUrl: './create-f3x-step1.component.html',
+})
+export class CreateF3XStep1Component implements OnInit, OnDestroy {
+  private destroy$ = new Subject();
+  formProperties: string[] = [
+    'filing_frequency',
+    'report_type_category',
+    'report_code',
+    'coverage_from_date',
+    'coverage_through_date',
+    'date_of_election',
+    'state_of_election',
+  ];
+  userCanSetFilingFrequency: boolean = environment.userCanSetFilingFrequency;
+  stateOptions: PrimeOptions = [];
+  formSubmitted = false;
+
+  form: FormGroup = this.fb.group(this.validateService.getFormGroupFields(this.formProperties));
+
+  /**  This different label list is 'necessarry' because the labels in the wireframe are
+   * different between the list and the creation steps
+   * */
+  f3xReportCodeCreationLabels: LabelList = [
```

Comment:
@mjtravers this is failing the sonarcloud test for code duplication because honestly it is pretty close to duplicate code.  The labels in the reports list are unfortunately different from the ones in the creation screens

Reply:
We'll need to investigate further a way to flag this as an exception or maybe increase the passing limit percentage for duplicate code.